OTKit/otkit-icons/ic_party_count.svg

```diff
@@ -0,0 +1,11 @@
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Icon/Line/ic_party_count" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path d="M16.2631227,13.2586903 C18.1508089,13.7307269 19.5546223,14.8487636 20.4745627,16.6128003 C20.6979132,17.0410874 20.8145462,17.5169728 20.8145462,18 C20.8145462,19.656863 19.4713933,21.0000158 17.8145304,21.0000158 L10.1861272,21 C9.70310501,21 9.22722453,20.8833688 8.79894144,20.6600214 C7.32985342,19.8938999 6.7599865,18.0819045 7.52610601,16.6128155 C8.44596957,14.8489171 9.84962812,13.7309148 11.7370816,13.2588084 C10.4715685,12.4922943 9.62582312,11.102268 9.62582312,9.51470731 L9.62582312,9.37418158 C9.62582312,6.9583878 11.5842109,5 14.0000047,5 C16.4157985,5 18.3741863,6.9583878 18.3741863,9.37418158 L18.3741863,9.51470731 C18.3741863,11.1021865 17.5285277,12.4921515 16.2631227,13.2586903 Z M14.0000047,6.77777778 C12.5505284,6.77777778 11.3754957,7.95281046 11.3754957,9.40228673 L11.3754957,9.48660216 C11.3754957,10.9360784 12.5505284,12.1111111 14.0000047,12.1111111 C15.449481,12.1111111 16.6245136,10.9360784 16.6245136,9.48660216 L16.6245136,9.40228673 C16.6245136,7.95281046 15.449481,6.77777778 14.0000047,6.77777778 Z M19.1407223,17.8834842 C18.3953349,15.8130133 16.6818457,14.7777778 14.0002549,14.7777778 C11.3186989,14.7777778 9.60510293,15.8129863 8.85946694,17.8834035 L8.85948629,17.8834105 C8.67235704,18.4030147 8.94188126,18.9759353 9.46148548,19.1630646 C9.57016378,19.2022038 9.68480169,19.2222174 9.80031295,19.2222174 L18.1998194,19.2222365 C18.7521147,19.2222365 19.1998386,18.7745127 19.1998386,18.2222174 C19.1998386,18.10674 19.1798375,17.9921351 19.1407223,17.8834842 Z" id="Combined-Shape" fill="#2D333F"></path>
+        <path d="M6.84758092,15.2868877 L4.95281574,15.2868904 C4.70769063,15.2868904 4.46465381,15.2418286 4.23582208,15.1539516 C3.2046777,14.7579669 2.68977901,13.6010502 3.08575512,12.5699026 C3.82069874,10.6561088 5.1320025,9.5117912 7.0196664,9.13694976 C5.82934918,8.60434475 5,7.4096537 5,6.02125777 L5,5.91165437 C5,4.02744969 6.52744969,2.5 8.41165437,2.5 C9.71370481,2.5 10.8453945,3.22940104 11.4204073,4.30188698 C11.0517254,4.51502578 10.7078278,4.76626177 10.3940966,5.05021297 C10.0643294,4.36191626 9.36117108,3.88658243 8.54699262,3.88658243 C7.41646981,3.88658243 6.5,4.80305224 6.5,5.93357505 L6.5,5.99933709 C6.5,7.1298599 7.41646981,8.04632971 8.54699262,8.04632971 C8.56352789,8.04632971 8.58001737,8.04613366 8.59645886,8.04574374 C8.48054072,8.51136745 8.41901325,8.99849847 8.41901325,9.5 C8.41901325,9.67501296 8.42650641,9.84827579 8.44118941,10.0194852 C6.51730233,10.1578931 5.25085251,11.0336588 4.64183995,12.6467822 L4.64182597,12.646777 C4.44675453,13.1634729 4.70748271,13.7404744 5.22417865,13.9355458 C5.33706205,13.9781634 5.45672639,14 5.57738674,14 L7.95514558,13.9999938 C7.52906888,14.3735851 7.15548041,14.8065681 6.84758092,15.2868877 Z" id="Combined-Shape" fill="#2D333F"></path>
```

Comment:
Use `ctrl + f` for `Combined-Shape` here... is this correct?

Looks like the build is failing because both of these have the same ID

Reply:
Ok I found the issue. Will update.